src/ngraph/frontend/onnx_import/op/clip.cpp

```diff
@@ -43,6 +49,54 @@ namespace ngraph
 
             } // namespace set_1
 
+            namespace set_11
+            {
+                NodeVector clip(const Node& node)
+                {
+                    NodeVector inputs{node.get_ng_inputs()};
+                    std::shared_ptr<ngraph::Node> data = inputs.at(0);
+                    const element::Type data_type = data->get_element_type();
+                    const Shape data_shape = data->get_shape();
+                    std::shared_ptr<ngraph::Node> min;
+                    std::shared_ptr<ngraph::Node> max;
+
+                    // If second input is provided, assign to min input, otherwise set lowest
+                    // numeric limit of double as min input.
+                    if (inputs.size() > 1 && !inputs.at(1)->is_null())
+                    {
+                        min = inputs.at(1);
+                    }
+                    else
+                    {
+                        min = builder::make_constant(
+                            data_type, data_shape, std::numeric_limits<double>::lowest());
+                    }
+
+                    // If third input is provided, assign to max input, otherwise set maximum
+                    // numeric limit of double as max input.
+                    if (inputs.size() == 3 && !inputs.at(2)->is_null())
+                    {
+                        max = inputs.at(2);
+                    }
+                    else
+                    {
+                        max = builder::make_constant(
+                            data_type, data_shape, std::numeric_limits<double>::max());
+                    }
+
+                    auto max_of_min_and_data = std::make_shared<ngraph::op::Maximum>(
+                        min,
+                        data,
+                        ngraph::op::AutoBroadcastSpec(ngraph::op::AutoBroadcastType::NUMPY));
+
+                    return {std::make_shared<ngraph::op::Minimum>(
+                        max,
+                        max_of_min_and_data,
+                        ngraph::op::AutoBroadcastSpec(ngraph::op::AutoBroadcastType::NUMPY))};
```

Comment:
Please remove these `ngraph::` prefixes as well.

Reply:
It can't be removed because of namespace conflicts.